apps/Gruntfile.js

```diff
@@ -21,6 +21,7 @@ module.exports = function (grunt) {
     // get compiled.
     fs.writeFileSync(
       'test/entry-tests.js',
+      "require('babel-polyfill');\n" +
```

Comment:
Why isn't this added to entry-tests.js itself - or above it, in case someone writes ES6 in that file?